SwiftLeeds/Data/Model/Sponsor.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  Sponsor.swift
+//  SwiftLeeds
+//
+//  Created by Alex Logan on 01/07/2022.
+//
+
+import Foundation
+
+// These should come from an API eventually
+struct Sponsor: Hashable {
+    let name: String
+    let oneLiner: String
+    let imageURL: URL?
+    let link: URL?
+}
+
+// MARK: - Static data
+extension Sponsor {
+    static let platinum = [codemagic, stream]
```

Comment:
As this value is not used, it should be removed

Reply:
As requested above, this is now used.

Reply:
Put this comment in as the above response said it wouldn't be used, but if it now is, makes sense.